conans/test/integration/graph/test_deferred.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient
+from conans.util.files import save
+
+
+class TestToolRequires:
+    def test_deferred(self):
+        client = TestClient()
+        client.save({"conanfile.py": GenConanfile().with_tool_requires("tool/1.0"),
+                     "profile": "[deferred_requires]\ntool/1.0"})
+        client.run("create . pkg/1.0@ -pr=profile")
+        assert "tool/1.0:357add7d387f11a959f3ee7d4fc9c2487dbaa604 - Deferred" in client.out
```

Comment:
How is the package id calculated for a deferred node?

Reply:
Same as others. As the package recipe is basically empty (no settings, no options, no dependencies), the package_id of the deferred will always be the same, we name it in our tests as: ``NO_SETTINGS_PACKAGE_ID = "357add7d387f11a959f3ee7d4fc9c2487dbaa604"``